portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,33 +36,56 @@
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   private static final String DEFAULT_COMMENTS_COUNT = "5";
   private static final String DEFAULT_PAGE = "1";
-  private static final String DEFAULT_COMMENT = "";
+  private static final String DEFAULT_SORT = "time-newest";
+
+  private static class Comment {
+    long timestamp;
+    String name;
+    String text;
```

Comment:
You can mark these as "final" if you don't want to modify them once initialized in the constructor.
Besides, if you don't want to access them directly, considering use "private" modifier.